include/internal/plat-msft.h

```diff
@@ -0,0 +1,166 @@
+/*
```

Comment:
I think you can remove this one now ;-)

Reply:
GitHub confusion?  I'll do another push/-f but I did 'git mv' the file.